x_xy/render.py

```diff
@@ -241,22 +242,50 @@ def __init__(
         else:
             self.disable_xyz(not show_cs_root)
 
-    def _get_camera(self) -> scene.cameras.BaseCamera:
-        return self.view.camera
-
     def _set_camera(self, camera: scene.cameras.BaseCamera) -> None:
         self.view.camera = camera
 
+    def _get_camera(self) -> scene.cameras.BaseCamera:
+        return self.view.camera
+
     def _render(self) -> jax.Array:
         return self.canvas.render(alpha=True)
 
-    def _add_box(self, geom: base.Box):
-        return scene.visuals.Box(
-            geom.dim_x,
-            geom.dim_z,
-            geom.dim_y,
+    def _add_box(self, box: Box) -> Visual:
+        return visuals.Box(
+            box.dim_x,
```

Comment:
Why not also here `vispy_visuals.Box` ?

Reply:
Because for unknown reasons vispy_visuals.Box does not support shading="smooth"

Reply:
That's a fair enough. Maybe include that as a comment somewhere that this Box is needed because the provided Box does not work because ...

Reply:
done